assets/data-port/export/export-progress-page.js

```diff
@@ -0,0 +1,89 @@
+import { Button } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { downloadFile } from '../../shared/helpers/download-file';
+import { Notice } from '../notice';
+
+export { __ } from '@wordpress/i18n';
+
+/**
+ * @typedef ExportProgressPageState
+ * @property {string} status  Export status.
+ * @property {number} percentage Export progress percentage.
+ * @property {ExportFile[]}   Exported files.
+ * @property {string} error Error message.
+ */
+/**
+ * @typedef ExportFile
+ * @property {string} url  File URL.
+ * @property {string} name File name.
+ */
+/**
+ * Exporter progress and result.
+ *
+ * @param {Object} props
+ * @param {ExportProgressPageState} props.state
+ * @param {Function} props.reset Function to return to initial export screen.
+ */
+export const ExportProgressPage = ( { state, reset } ) => {
+	const { status, percentage, files, error } = state || {};
+
+	const inProgress = 'completed' !== status;
+
+	useEffect( () => {
+		if ( ! files ) return;
+
+		files.forEach( downloadFile );
+	}, [ files ] );
+
+	return (
+		<section className="sensei-data-port-step__body">
+			{ inProgress ? (
+				<p>
+					<progress
+						className="sensei-data-port__progressbar"
+						max="100"
+						value={ percentage || 0 }
+					/>
+				</p>
+			) : (
+				<>
+					<div className="sensei-export__output-result">
+						{ files && (
+							<>
+								<p>
+									{ __(
+										'The following files were exported:',
+										'sensei-lms'
+									) }
+								</p>
+								<div className="sensei-export__output-files">
```

Comment:
Could we use a `ul` here to be more semantic?

Reply:
e1f53a2